src/common/types/core.js

```diff
@@ -131,14 +130,14 @@ type MarkerPaginatedCollection<T> = {
     next_marker: ?string,
 };
 
-type SelectorItem = {
+type SelectorItem<T: any = any> = {
```

Comment:
Changed this to a generic so that we can tell what the type of the items are, as we will be using that information to change the display of the selector UI

Reply:
We noticed yesterday that the existing `SelectorItem` entries have the UserMini details merged at the top layer instead of under item. For existing functionality, item only contained `text` and `value`.